terraform/workspace-variables/dev_app_env.yml

```diff
@@ -0,0 +1,9 @@
+---
+RACK_ENV: production
+RAILS_ENV: deployed_development
+NODE_ENV: production
+RAILS_LOG_TO_STDOUT: true
+RAILS_MAX_THREADS: 5
+RAILS_SERVE_STATIC_FILES: true
```

Comment:
I think any reference to rails can be deleted, this looks like leftovers from pinching the ECF stuff